experiments/acic18/script.R

```diff
@@ -3,6 +3,7 @@ set.seed(1)
 rm(list = ls())
 
 library(grf)
+if(packageVersion("grf") >= '0.10.2') {warning("This script requires grf 0.10.2 or higher");}
```

Comment:
I think this should be a '<' rather than '>='

Reply:
I totally agree.